src/components/common/ParentFooter.js

```diff
@@ -37,6 +35,7 @@ export default function ParentFooter(props) {
             </Link>
           </div>
         </nav>
+        <div class="copyright">©{curYear} Story Squad HQ</div>
```

Comment:
Nice simple fix to clean up the layout of the footer!

Reply:
Thanks Paul.